spec/support/helpers/wordpress_api_helper.rb

```diff
@@ -0,0 +1,132 @@
+module WordpressApiHelper
```

Comment:
I'd avoid the name `Helper` to avoid confusions with Rails Helpers that are intended to be used in views. WDYT about `WordpressMocker` ?

Reply:
Sure! What about `WordpressAPIMocker`?

Reply:
sure